### ISSUE TITLE
unable to reproduce results (ResNet-50, Galaxy10)

### ISSUE THREAD

**Issue by elastufka** [pranavphoenix/Backbones#1]
Hi, thanks for doing all these comparisons!

I was trying out the code and decided to reproduce your result from Table 5 for ResNet-50 on Galaxy10 (top-1 accuracy =  84.80). I initialized the network with the PyTorch pre-trained weights.

Granted, I just used one GPU so the batch size was 64 instead of 256 but I did do gradient accumulation to make up for that. After training with the ADAMW optimizer, top-1 accuracy was 0.81. Training further with SGD did nothing to improve it.

Was there perhaps anything done that was not mentioned in the code? The code seems like it might not be the final version. Also, on that dataset it is quite common to do a significant center crop to something less than 200x200.
